apiserver/facades/agent/uniter/service.go

```diff
@@ -65,8 +65,12 @@ type ApplicationService interface {
 // UnitStateService describes the ability to retrieve and persist
 // remote state for informing hook reconciliation.
```

Comment:
Related to comments about UnitState, not necessarily this PR:

We're using the term remote state which has a specific meaning in the uniter. I saw a comment above, "the state internal to the uniter for this unit" which is more accurate. Changes to this state will not trigger an event hook unless the unit restarts in the middle of a hook or such.

Reply:
I've changed "remote state" to "unit agent state".

Reply:
The uniter is only part of the unit agent. This data does not encompass the unit agent state. `Unit state` would be more accurate.